plugin/JavaImp.vim

```diff
@@ -203,21 +203,28 @@ function! <SID>JavaImpAppendClass(cpath, relativeTo)
             call append(0, a:relativeTo)
         endif
     elseif (isdirectory(a:cpath))
-		" Recursively fetch all Java files from the provided directory path.
-        let l:javaList = glob(a:cpath . "/**/*.java", 1, 1) 
-		let l:clssList = glob(a:cpath . "/**/*.class", 1, 1) 
-		let l:list = l:javaList + l:clssList
-
-		" Include a trailing slash so that we don't leave a slash at the
-		" beginning of the fully qualified classname.
-		let l:cpath = a:cpath . "/"
+			" Recursively fetch all Java files from the provided directory path.
```

Comment:
There are some whitespace issues between lines 207 and 227.  For this project, can you prefer 4 spaces over a tab for indentation?